environments/prod/cjscp-org-uk.yml

```diff
@@ -419,18 +419,6 @@ cname:
   - name: s2._domainkey
     ttl: 600
     record: s2.domainkey.u10165397.wl148.sendgrid.net
-  - name: _C16918C74EEC65396149B5A9FA7570BC.login
-    ttl: 1800
-    record: 66146904CA2BA454CFB1C6B5DE7EC167.689BC6A211760B22BB63B282EA5B7F11.b420a0d77f3ec009704f.sectigo.com.
-  - name: _C16918C74EEC65396149B5A9FA7570BC.identity-gateway
-    ttl: 1800
-    record: 66146904CA2BA454CFB1C6B5DE7EC167.689BC6A211760B22BB63B282EA5B7F11.b420a0d77f3ec009704f.sectigo.com.
-  - name: _C16918C74EEC65396149B5A9FA7570BC.manage-users
-    ttl: 1800
-    record: 66146904CA2BA454CFB1C6B5DE7EC167.689BC6A211760B22BB63B282EA5B7F11.b420a0d77f3ec009704f.sectigo.com.
-  - name: _C16918C74EEC65396149B5A9FA7570BC.my-services
-    ttl: 1800
-    record: 66146904CA2BA454CFB1C6B5DE7EC167.689BC6A211760B22BB63B282EA5B7F11.b420a0d77f3ec009704f.sectigo.com.
-  - name: _C16918C74EEC65396149B5A9FA7570BC.token
-    ttl: 1800
-    record: 66146904CA2BA454CFB1C6B5DE7EC167.689BC6A211760B22BB63B282EA5B7F11.b420a0d77f3ec009704f.sectigo.com.
+  - name: _74BA1C682E94EF85F92231FD5466DBA8.prosecuting
+    ttl: 10800
```

Comment:
ttl is 10800

Reply:
Yeah, that's the number mentioned in the temp records email.